packages/main/src/TabContainer.ts

```diff
@@ -975,7 +989,7 @@ class TabContainer extends UI5Element {
 		const tabStrip = this._getTabStrip();
 		let allItemsWidth = 0;
 
-		if (!this._selectedTab) {
+		if (!this._selectedTab && !this.noDefaultSelection) {
 			return;
 		}
```

Comment:
It's safe to remove this statement completely

Reply:
Without it, the overflow behavior breaks